### ISSUE TITLE
The updating of mask

### ISSUE THREAD

**Issue by chenxiaodanhit** [NVIDIA/partialconv#41]
Thank you for your great contributions! May i ask you for the updating of the mask in partial convolution?  The mask and weight_maskUpdater are initialized by torch.ones and the results of  update_mask are always identity matrix. Thank you for your help!

**Comment by ivanstepanovftw** [comment]
It is not an identity matrix, it is a matrix full of ones.
